frontend/src/modules/WellLogViewer/view/SubsurfaceLogViewerWrapper.tsx

```diff
@@ -0,0 +1,265 @@
+    const [wellLogReadout, setWellLogReadout] = React.useState<Info[]>([]);
+    const [showReadoutBox, setShowReadoutBox] = React.useState<boolean>(false);
+
+    const { template, welllog, wellpicks } = useViewerDataTransform(props);
+
+    const colorScale = props.moduleProps.workbenchSettings.useContinuousColorScale({
+        gradientType: ColorScaleGradientType.Sequential,
+    });
+    const colorTables = React.useMemo(() => createContinuousColorScaleForMap(colorScale), [colorScale]);
+
+    // Global value syncronization
+    const broadcastGlobalMdChange = useGloballySyncedMd(
+        props.wellboreHeader,
+        wellLogController,
+        props.moduleProps.workbenchServices,
+        props.moduleProps.viewContext
+    );
+
+    const broadcastVerticalScaleChange = useGloballySyncedVerticalScale(
+        wellLogController,
+        props.moduleProps.workbenchServices,
+        props.moduleProps.viewContext
+    );
+
+    const handleMouseOut = React.useCallback(
+        function handleMouseOut() {
+            broadcastGlobalMdChange(null);
+            setShowReadoutBox(false);
+        },
+        [broadcastGlobalMdChange]
+    );
+
+    const handleMouseIn = React.useCallback(function handleMouseIn() {
+        setShowReadoutBox(true);
+    }, []);
+
+    // Log viewer module callbacks
+    const handleCreateController = React.useCallback(function handleCreateController(controller: WellLogController) {
+        // ? Something weird happens during HMR, where the controller ref becomes null, but this event still fires?
+        console.debug("Setting well log viewer controller...", controller);
+        setWellLogController(controller);
+    }, []);
+
+    const handleContentRescale = React.useCallback(
+        function handleContentRescale() {
+            const currentScale = wellLogController?.getContentZoom();
+
+            if (currentScale) broadcastVerticalScaleChange(currentScale);
+        },
+        [broadcastVerticalScaleChange, wellLogController]
+    );
+
+    const handleSelection = React.useCallback(
+        function handleSelection() {
+            const currentSelection = wellLogController?.getContentSelection()[0] ?? null;
+
+            broadcastGlobalMdChange(currentSelection);
+
+            // TODO: It's possible to pin and select a range, should we have that color a section of other synced intersections?
+        },
+        [broadcastGlobalMdChange, wellLogController]
+    );
+
+    const handleInfoFilled = React.useCallback(function handleInfoFilled(infos: Info[]) {
+        setWellLogReadout(infos);
+    }, []);
+
+    const handleTrackMouseEvent = React.useCallback(
+        (/* welllogView: WellLogView, e: TrackMouseEvent */) => {
+            // ! No-op method. Passed to the viewer to make it not show the context menu for tracks
+        },
+        []
+    );
+
+    return (
+        // The weird tailwind-class hides the built-in hover tooltip
```

Comment:
Any chance to implement that as a feature in the `WellLogViewer`?

Reply:
Could be nice. Leaving the css here for now, but added an issue in the component library git; https://github.com/equinor/webviz-subsurface-components/issues/2390